drivers/flash/flash_stm32f4x.c

```diff
@@ -51,7 +51,7 @@ static int write_byte(const struct device *dev, off_t offset, uint8_t val)
 		return rc;
 	}
 
-	regs->CR &= ~CR_PSIZE_MASK;
+	regs->CR &= CR_PSIZE_MASK;
```

Comment:
This could have been written as `regs->CR &= ~(FLASH_CR_PSIZE)`  to use definitions coming directly from MCU specific include files, but since they are all the same it looks fine.